.github/workflows/rust.yml

```diff
@@ -28,6 +28,7 @@ jobs:
         run: cargo clippy
 
       - name: Build
+        continue-on-error: true
```

Comment:
I'd have gone with an arg making this optional instead of going with continue-on-error, but this is also a good option. Just mentioning, because this specifically wasn't my idea, contrary to the PR description.